tetraelf-hierarchy/hierarchy-api/api/src/java/org/sakaiproject/hierarchy/api/PortalHierarchyService.java

```diff
@@ -211,6 +211,13 @@
 	 * @return <code>true</code> if the node can have it's site changed.
 	 */
 	boolean canChangeSite(String id);
-	
-	
+
+	/**
```

Comment:
Because there is a PortalHierarchyService object in HierarchyEntityProviderImpl but not a SiteService object.